security/api/src/main/java/org/jboss/as/security/api/ConnectionSecurityContext.java

```diff
@@ -45,7 +45,7 @@
  *
  * As a connection is established to the application server the remote user is authenticated, this API allows the
  * {@link Collection} of {@link Principal}s for the remote user to be obtained, the API then allows for an alternative identity
- * to be pushed by interceptors for validation in the security interceptors for subsequent EJB invocations.
+ * to be pushed by Jakarta Interceptors for validation in the security Jakarta Interceptors for subsequent EJB invocations.
```

Comment:
I suspect the interceptors here are something different from the JakartaEE interceptors